db/migrate/20220705164609_add_user_ref_to_comment.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] According to the ERD, the foreign key in the comments table references the `author` in the Users table. Hint 👇 

```suggestion
  def change
    add_reference :comments, :author, foreign_key: { to_table: :users }, null: false
  end
```